recipes/pedesigner/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pedesigner" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "67d1d854227ea1868da9ae17580e034b0bc3dad0445e282860558dc3bd52b27d" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: python
+
+source:
+  url: https://github.com/VeredKunik/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - python >=3.7
+
+  run:
+    - python >=3.7
+    - cas-offinder
+
+test:
+  imports:
+    - pedesigner
+  commands:
+    - pedesigner --help
+
+about:
+  home: https://github.com/VeredKunik/pedesigner
+  license: GPL3
```

Comment:
Can you please use spdx license identifiers.

Reply:
Yes, it's the wrong file altogether. I am updating it.
Apologies!